tensorly/backend/pytorch_backend.py

```diff
@@ -35,6 +35,10 @@ def context(tensor):
 
     @staticmethod
     def tensor(data, dtype=torch.float32, device="cpu", requires_grad=False):
+        if isinstance(data, torch.Tensor):
```

Comment:
This is just to silence a warning from PyTorch about not abusing `torch.tensor`.